src/Makefile.am

```diff
@@ -66,6 +66,7 @@ pd_SOURCES_core = \
     g_hdial.c \
     g_hslider.c \
     g_io.c \
+    g_kbdnav.c \
```

Comment:
as mentioned before: `g_kbdnav.h` must be added to `noinst_HEADERS`.
And you must include the compilation of `g_kbdnav.c` in `src/maefile.*`